src/utils/bridge/eth/history.ts

```diff
@@ -0,0 +1,351 @@
+import flatten from 'lodash/fp/flatten';
+import first from 'lodash/fp/first';
+import last from 'lodash/fp/last';
+import { ethers } from 'ethers';
+import { BridgeTxStatus, Operation } from '@sora-substrate/util';
+import {
+  api,
+  historyElementsFilter,
+  SubqueryExplorerService,
+  SUBQUERY_TYPES,
+  WALLET_CONSTS,
+} from '@soramitsu/soraneo-wallet-web';
+
+import ethersUtil from '@/utils/ethers-util';
+import { getEvmTransactionRecieptByHash, isOutgoingTransaction } from '@/utils/bridge/common/utils';
+
+import { ethBridgeApi } from '@/utils/bridge/eth/api';
+
+import { ZeroStringValue } from '@/consts';
+
+import type { Asset } from '@sora-substrate/util/build/assets/types';
+import type { BridgeHistory, NetworkFeesObject } from '@sora-substrate/util';
+import type { EthBridgeContractsAddresses } from '@/store/web3/types';
+
+const BRIDGE_ABI = new ethers.utils.Interface([
+  'function mintTokensByPeers(address tokenAddress,uint256 amount,address beneficiary,bytes32 txHash,uint8[] v,bytes32[] r,bytes32[] s,address from)',
```

Comment:
it's not good to store ABI as a string I think cuz it might cause issues... we need to think about alternatives here